docs/guides/cloud/branch-review.mdx

```diff
@@ -0,0 +1,178 @@
+<DocsImage
+  src="/img/guides/cloud/branch-review/labs-beta.png"
+  alt="Enable Branch Review Beta"
+  width="90%"
+/>
+
+:::
+
+## Getting Started
+
+Branch Review works by leveraging the Cypress Cloud GitHub integration to query the GitHub API for branches with pull requests. Your project will first need to be connected to a GitHub repository. If you haven't already, follow the steps in [Install the Cypress GitHub app](/guides/cloud/integrations/source-control/github#Install-the-Cypress-GitHub-app) to connect your project to GitHub.
+
+### Accessing Branch Review
+
+To access Branch Review, navigate to the **Branches** tab in the left sidebar and select the branch you want to review. In order to see a comparison, the branch you select must have a pull request.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/get-to-branch-review.jpg"
+  alt="Get to Branch Review"
+  caption="Active branches, shown with available PR # tags"
+/>
+
+If you do not see PR # tags associated with a branch, you may need to [pass PR numbers](/guides/continuous-integration/introduction#CI-Build-Information) to runs as environment variables. This helps ensure Cypress Cloud can map PR data correctly.
+
+#### Latest Runs
+
+If the selected branch has a PR associated and a PR # passed via CI, there is an additional callout on the top of the overview tab of the **Latest runs** page directs to the Branch Review.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/branch-review-available.jpg"
+  alt="Branch Review available banner"
+/>
+
+## Branch Details
+
+### View Options
+
+"Review" is the default view of the **Branches** tab for projects with linked GitHub repositories via our GitHub integration.
+
+"Runs" provides a list filtered by the selected branch for projects without linked repositories or those with an unknown branch. An unkown branch is one in which Cypress does not have access to the local `.git` directory and cannot determine a branch for a run.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/branch-review-views.jpg"
+  alt="Displaying different views for branches"
+/>
+
+### Review Header
+
+The header includes the Git commit message, PR selector, PR status, and base and feature branch labels with the Cypress Cloud test run ID number (#).
+
+If there are multiple pull requests open for the same branch, you can select the pull request you want to review from the dropdown to the right of the commit message.
+
+Clicking the branch label will link you directly to the appropriate [run overview](/guides/cloud/debugging/recorded-runs#Overview-tab) and hovering on these elements exposes additional run meta data.
+
+<DocsImage
+  src="/img/guides/cloud/branch-review/branch-review-header.jpg"
+  alt="Branch Review Header"
+/>
+
+### Review Test Status
+
+When on the review screen, you will see tabs Failures, Flaky, Pending, Added, and Modified. Each tab will show you the specs that fall into that category. You can click on a test to view the test details.
```

Comment:
Grammar edit:
When on the review screen, you will see THE Failures, Flaky, Pending, Added, and Modified TABS. Each tab will show you the specs that fall into that category. You can click on a test to view the test details.